civihr_default_theme/scss/partials/_forms.scss

```diff
@@ -30,17 +30,6 @@
 // Theme has issues with ui-select search field
 // @TODO remove these styles below as soon as Shoreditch theme is applied
 
-.ui-select-search {
-  width: 100% !important;
-}
-
-.ui-select-multiple {
-
-  .ui-select-search {
-    width: 150px !important;
-  }
-}
-
 .ui-select-toggle {
```

Comment:
what about this?

Reply:
Yes it is also not required, the default value of `ui-select-toggle` is also `7px 10px`is SSP.